.github/workflows/serving_publish.yml

```diff
@@ -58,17 +58,23 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-2
-      - name: Publish snapshot deb
+      - name: Publish snapshot deb, tar and zip
         if: ${{ github.event.inputs.mode == '' || github.event.inputs.mode == 'snapshot' }}
         run: |
+          ./gradlew :serving:distZip -Psnapshot
```

Comment:
We don't need publish snapshot .zip and .tar file

Reply:
Fixed.